server/controllers/Contacts.js

```diff
@@ -63,7 +63,7 @@ module.exports.addContact = async (req, res) => {
 };
 
 module.exports.getContacts = async (req, res) => {
-    let userId = req.params.userId;
+    let userId = req.user.id;
```

Comment:
Comment below can be removed because we no longer need to add that check since the user id comes from the jwt